release/rllib_tests/learning_tests/yaml_files/ppo/old_stack/ppo-breakoutnoframeskip-v5.yaml

```diff
@@ -24,6 +24,9 @@ ppo-breakoutnoframeskip-v5:
         sgd_minibatch_size: 500
         num_sgd_iter: 10
         num_workers: 30
+        # Run with Learner- and RLModule API (new stack).
+        _enable_learner_api: false
+        _enable_rl_module_api: false
```

Comment:
This makes is so that the old stack test acutally tests the old stack.

Reply:
Typo: Run without ??